slicingTool/runProjectSlicingTool.sh

```diff
@@ -17,7 +17,15 @@ RUN_CHAR_FIX=$(grep runCharacterFixer slicingTool.prop | grep -v '\#' |perl -pe
 #    exit
 #fi
 
-if [[ $RUN_CHAR_FIX == "true" ]] ; then
+echo -n "Would you like to run the version topic comparison between this release and last? (y/n): "
+read ver_topic
+if [[ $ver_topic == y* ]] || [[ $ver_topic == Y* ]]
```

Comment:
why not just match "y" exactly? That's what's in the question.

Reply:
I just wanted the script to flexible enough to understand in case the user types "Y", "yes", "Yes", etc. instead of exactly "y".

Reply:
I am not sure that level of flexibility is really necessary, but OK.